doctoshotgun.py

```diff
@@ -228,19 +232,26 @@ def do_login(self):
 
         return True
 
-    def find_centers(self, where, motives=None):
+    def find_centers(self, where, motives=None, currentPage=1):
```

Comment:
All other function attributes are in snake_case. Let's keep things consistent and name it `current_page`.